timeout 43200 /usr/bin/python /home/coffeyrt/311pull/311pull.py
/usr/bin/python /home/coffeyrt/311pull/311plot.py --database /media/main/311.db --out /media/main/stock/
